packages/js/client/src/__tests__/core/resolveUri.spec.ts

```diff
@@ -1,57 +1,34 @@
+import { Uri } from "../..";
+import { buildWrapper } from "@polywrap/test-env-js";
+import { GetPathToTestWrappers } from "@polywrap/test-cases";
 import {
-  createPolywrapClient,
-  Uri,
-  PolywrapClientConfig,
-  PolywrapClient,
+  coreInterfaceUris,
   PluginModule,
-} from "../..";
-import {
-  buildAndDeployWrapper,
-  initTestEnvironment,
-  runCLI,
-  stopTestEnvironment,
-  ensAddresses,
-  providers
-} from "@polywrap/test-env-js";
-import { GetPathToTestWrappers } from "@polywrap/test-cases";
-import { ResolveUriErrorType } from "@polywrap/core-js";
+  ResolveUriErrorType,
+} from "@polywrap/core-js";
+import { getClient } from "../utils/getClient";
 
 jest.setTimeout(200000);
 
-describe("resolveUri", () => {
-  let ipfsProvider: string;
-  let ethProvider: string;
+const wrapperPath = `${GetPathToTestWrappers()}/wasm-as/simple`;
+const wrapperUri = new Uri(`wrap://file/${wrapperPath}/build`);
 
-  beforeAll(async () => {
-    await initTestEnvironment();
-    ipfsProvider = providers.ipfs;
-    ethProvider = providers.ethereum;
-  });
+const simpleFsResolverWrapperPath = `${GetPathToTestWrappers()}/wasm-as/simple-fs-resolver`;
+const simpleFsResolverWrapperUri = new Uri(
+  `wrap://file/${simpleFsResolverWrapperPath}/build`
```

Comment:
Just curious here: Is there any reason to use `file` instead of `fs`?

Reply:
I think we should slowly start to faze out `fs` and instead use the standard `file://` scheme name.

Reply:
> Just curious here: Is there any reason to use `file` instead of `fs`? I think that indeed we should change it from `fs` to `file` but probably we should keep consistency for now?

`file` is just more commonly used in file URL outside of polywrap. E.g. in the browser, or package.json when pointing to local packages

Reply:
> I think we should slowly start to faze out fs and instead use the standard file:// scheme name.

I would suggest opening a PR and changing this in every part of the codebase (ASAP). We know that `fs` and `file` are equivalent in our codebase, but this might cause some confusion to people that check our codebase IMO